tests/unit/certGen.unitTest.ts

```diff
@@ -2143,7 +2148,7 @@ describe("cert-gen", () => {
                 it("should bubble that error up", async () => {
                     const event: any = {Records: [{...queueEvent.Records[0]}]};
 
-                    sinon.stub(CertificateUploadService.prototype, "uploadCertificate").throws(new Error("It broke"));
+                    sandbox.stub(LambdaService.prototype, "invoke").throws(new Error("It broke"));
```

Comment:
This test was broken. Changed to fix. Doesn't look like tests were being run on Cert Gen previously

Reply:
Sandbox change was also required to prevent conflicts with second test file.